processes/org-monthly-meetings/minutes/2019-07-Minutes.md

```diff
@@ -0,0 +1,114 @@
+# Organizers meeting minutes
+
+**DATE:** 2019-07-11
+
+**PARTICIPANTS:**
+_In alphabetical order_
+
+* @amaliacardenas
+* @daverick
+* @jmbriano
+* @loriking
+* @raquelalvear
+* @saljuama
+
+
+**LOCATION:**
+
+ThoughtWorks
+
+# Agenda
+
+## Ongoing activities (xx')
+
+|**Initiative**| Presented by |minutes|
+|:------------|--------------|:--:|
+|Specific workshops/Curriculum development|@loriking|x|
+|Define process for selecting next hosts|@amaliacardenas|x|
+|Codebar landing page. Explanation in 3 languages|@jmbriano|x|
+|Adding collaborators board|@jmbriano|x|
+|Pride Month|@daverick|x|
+|Handbook for organisers|@amaliacardenas|x|
+|Instagram account|@raquelalvear|x|
+|Adding a community manager|@loriking|x|
+
+
+## New initiatives (xx')
+
+|**Initiative**| Presented by |minutes|
+|:------------|--------------|:--:|
+||||
+
```

Comment:
```suggestion
|comunícate the minutes of our org meetings in the Bcn channel to promote interest in the org and have some initiative from our members|@daverick||
|a third meeting by month?|@daverick||
|a kit for coach so that they can explain to their coworker why coaching at codebar is super cool ( came from a chat with Alyssa from typeform to how to foment their employee to coach at codebar when it will be at typeform)|@daverick||
| BCN codebar role in building LGBTQIA + tech community|@daverick||
|Working with some student on making better the feedback from student to coach|@daverick with Edu||
|reuse TW Computing fondamental|@daverick||

```